cmd/clair/main.go

```diff
@@ -201,12 +203,36 @@ func Boot(config *Config, slimMode bool) {
 	serv.Close()
 }
 
+func bootNodeInventoryScanner() {
+	// TODO: Check: Unify with Scanner boot and run as a different configuration rather than a different function.
```

Comment:
Nit: Not clear what this TODO means. If we use the same config for both the regular scanner and `NodeInventory=true` scanner, will this still be necessary?

Maybe better to create a follow-up task instead of a TODO in the code.

Reply:
Removed the TODO. Your comment above about starting node inventory mode after setting up default config already solves this question.